packages/idx/src/idx.d.ts

```diff
@@ -34,18 +34,24 @@ type FunctionWithRequiredReturnType<
 type DeepRequired<T> = T extends any[]
   ? DeepRequiredArray<T[number]>
   : T extends (...args: any[]) => any
-    ? FunctionWithRequiredReturnType<T>
-    : T extends object ? DeepRequiredObject<T> : T;
+  ? FunctionWithRequiredReturnType<T>
+  : T extends object
+  ? DeepRequiredObject<T>
+  : T;
```

Comment:
I didn't change these lines but husky pre-commit hook formatted this file.

https://github.com/facebookincubator/idx/blob/master/package.json#L24